### ISSUE TITLE
Dupe Bug with Crafting Grid

### ISSUE THREAD

**Issue by Lumindia** [raoulvdberge/refinedstorage#210]
When shift-clicking the output slot with more than an item in a stack in the crafting grid, it 'removes' one item from the grid, but then produces as many items as were in the original stack. 

So a stack of, say, diamonds, would remove one diamond from the grid, but produce an additional 64.

This will not work if you only have one of the item, so items that cannot stack aren't affected by this as far as I know.

Happens in mod version 0.8.10 on Minecraft 1.10

**Comment by raoulvdberge** [comment]
I don't understand this bug report. How does it only get 1 item from a stack when shift clicking? Can you rephrase a bit and make it more clear?

**Comment by raoulvdberge** [comment]
Does this also happen for items that HAVE an output? E.g. does it dupe when shift clicking to make a workbench for example?

**Comment by Lumindia** [comment]
It does not appear to do so with recipes that have an output.

**Comment by raoulvdberge** [comment]
Thank you for reporting. See, this is an example of a bug that I would never have found myself
